### ISSUE TITLE
MiniGPT-4 support

### ISSUE THREAD

**Issue by denissabramovs** [hwchase17/langchain#3219]
Any plans to add it into the list of supported backend models?

**Comment by mortium91** [comment]
Yeah that would be great! We currently did some tests with MiniGPT4 and it seems to be a great asset
